.gitignore

```diff
@@ -10,6 +10,7 @@
 
 # Build System
 out/
+/examples/virtual-device-app/android/App/buildSrc/build/
```

Comment:
this change seems unrelated.

Reply:
Yes, it's unrelated to the code coverage itself, but when building tests my git tree is dirty because of this directory being created. I can create a separate PR for that...